src/App/Fossa/Configuration.hs

```diff
@@ -0,0 +1,99 @@
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module App.Fossa.Configuration
+( mergeFileCmdMetadata
+, readConfigFileIO
+, readConfigFile
+, ConfigFile (..)
+, ConfigProject (..)
+, ConfigRevision (..)
+)
+where
+
+import Data.Aeson ( FromJSON(parseJSON), (.:), (.:?), withObject )
+import Data.Text (Text)
+import App.Types
+import Path
+import qualified Control.Carrier.Diagnostics as Diag
+import Effect.ReadFS
+import System.Exit (die)
+import Options.Applicative
```

Comment:
[nit]: Make this import more clear (I couldn't figure it out without using vscode lens)
```suggestion
import Control.Applicative ( Alternative ((<|>)) )
```